lib/Hash/Fold/Error.pm

```diff
@@ -3,6 +3,10 @@ package Hash::Fold::Error;
 use Moo;
 extends 'Throwable::Error';
 
+# XXX this declaration must be on a single line
+# https://metacpan.org/pod/version#How-to-declare()-a-dotted-decimal-version
+use version 0.77; our $VERSION = version->declare('v1.0.0');
```

Comment:
Why does this need a version?

Reply:
I version every module in a distribution with the same value so that its possible to track the version of the distribution its associated with.  It does require one to keep them in sync. 

I used to use [ppi_version](https://metacpan.org/pod/PPI::PowerToys), but nowadays  I use [Dist::Zilla](https://metacpan.org/pod/Dist::Zilla), which can provide automatic version bumping and can generate docs from templates that put the version in the docs, so it's more easily accessible to the end user.

Reply:
There are cases where separate versions might be useful (I do it [here](https://github.com/chocolateboy/wax/commit/3e8a1051dc8306a78f0275e27ae139c6366be74b) for instance), but I don't think it's necessary or useful in this case, particularly as the versions aren't actually distinct and likely never will be.

Reply:
Thanks for clarifying things.